### DISCUSSION TITLE
Unable to switch to back facing camera on my android phone to take a picture of the document from my screen!

### DISCUSSION THREAD

**Discussion by AJV009** [community/community#49360] [GitHub Education] [answered]
You removed the upload button for safety, I'll take that.
But what's with a buggy camera interface for phones?
Imagine being stuck with a front-facing camera AND then holding it to the laptop screen which obviously has a NOT so good camera.
And then failing with a notice that I am not on campus? :exploding_head: 
I am in Maharashtra and the main Campus is Delhi, which is 1300 km away!
I am doing distance studies to meet ends.
Are you gonna fix these weird security requests like being on campus? Sure I see that you might have enough applicants from campuses implying that you don't care about students from home or distance studies.

**Comment by pewf** [comment]
This is happening to me right now, I can't finish my application because of this and it's impossible to get a good screen.

**Reply by hw88hw88** [reply]
I have the same issue.
Also, all my uploaded documents have been rejected, although any one of them is sufficient to prove my student identity and status.

**Comment by AJV009** [comment, marked as answer]
Hey everyone it took me some time to get the best picture but it was a success.
If like you have a digital ID and not a physical one (Which many have since the BIG COVID break)

Lower your screen brightness just enough to make it look like a book under the streetlight.
Hold your phone with both hands, while your elbows rest on the table.
Take a picture as soon as your ID is fits your phone screen.
The requirements are something like school name, your name year if any OR some identification that you are a distance learner if any should be required, a slightly blurry picture is fine to go.
Whatever filtering method @github is using can sharpen the image and extract text, and you will get an email within mins.
GitHub seems to be trying there best to make the process as automated as possible while also keeping fake accounts away from exploiting these premium resources.

**Reply by pewf** [reply]
Tips for how I would do this for digital transcript?

**Reply by P10swati** [reply]
I'm trying with my College library card ...will it work ?

**Comment by IAmTahazzot** [comment]
There are two option while taking a photo, 

Camera 2 1, facing front ✅ Works
Camera 2 0, facing back ❌ Doesn't work

It's nearly impossible to take documents photo with facing front camera. Switching to Camera 2 0, facing back doesn't work, it just stay in front camera.

**Reply by P10swati** [reply]
Try applying from your college campus if possible , document proof won't be an issue then , verification will be done using location.

**Reply by piincher** [reply]
Did you find any solution to it

**Reply by IAmTahazzot** [reply]
I've grown increasingly frustrated with the lack of attention given to this issue affecting students. 

It's clear that the problem is known, yet it remains unresolved, even on one of the most prominent software platforms like this for months. Regrettably, it doesn't seem likely to be fixed anytime soon.

Good luck, @piincher 
If you happen to succeed, please let us know the steps or strategies you employed.

**Reply by piincher** [reply]
What I did was to sit down with my admission notice in hand and then try to take a picture that is somewhat readable on the camera. After 2 minutes, it was approved

**Reply by IAmTahazzot** [reply]
With webcam or phone camera?

I tried both of it, failed many times.

**Reply by piincher** [reply]
Using phone camera

**Reply by IAmTahazzot** [reply]
Thanks. Btw did they fix the front-camera and back-camera changing option? or you did it with front-camera somehow?

**Reply by piincher** [reply]
The back camera problem is still there,I did with the front camera

**Reply by IAmTahazzot** [reply]
@piincher Which proof did you selected?

1. Dated school ID - Good
2. Dated official/unofficial transcript - Fair
3. Dated enrollment letter on school letterhead - Fair
4. Dated class schedule for the semester - Poor
5. Dated syllabus for a class - Poor
6. Dated receipt from bursar - Poor
7. Dated scholarship/financial aid letter - Poor
8. Other (Example: Screenshot of school portal) - Poor

I just wanna try exactly what you did, that's why. 
Thanks for helping.

**Reply by piincher** [reply]
the number 3

**Comment by piincher** [comment]
Hi GitHub Support @github

 I wanted to bring to your attention a bug I've encountered in the GitHub Education part of the platform. When I reach the stage where I'm required to take a picture of proof, I've noticed that the back camera doesn't work. Only the front camera seems to be functional.

This presents a significant challenge as it's often difficult to capture a suitable picture of proof using the front camera. I believe this issue is affecting the usability of the platform for many users like myself.

I kindly request that you investigate and address this issue as soon as possible to ensure a smoother experience for all GitHub Education users.

Thank you for your attention to this matter.

Best regards,
Regards

**Reply by IAmTahazzot** [reply]
@piincher You applied for it, right? `pro` badge shows in your profile. Can you help me? or are you on a `pro` plan?

**Reply by piincher** [reply]
What I did was to sit down with my admission notice in hand and then try to take a picture that is somewhat readable on the camera. After 2 minutes, it was approved

**Reply by piincher** [reply]
What I did was to sit down with my admission notice in hand and then try to take a picture that is somewhat readable on the camera. After 2 minutes, it was approved

**Comment by AJV009** [comment]
Looks like I am banned, I just cracked my personal savings to get some resources for a project.
GitHub Education no longer works for me. I tried a ton of things. Being a 21yr old student with no GH Education Access is fun :joy: Seeing the resources being provided for free, makes me push harder and harder to earn something and more!

**Comment by SamarthRajj** [comment]
Well it is 11 jan 2023 the bug is still not fixed and my phone can't take clear photos by front camera. I am on my vacations at home but it says i am too far from the college. Even if i go to college adter break how will i take picture with front  camera of my 10000 rupee smart phone.

**Comment by leventkeles** [comment]
in March 2024 the bug still not fixed, @github please fix those problems. We can not upload any document also we can not use back camera :( It is too hard to prove document with front camera, I have no student card, only document.

**Reply by cesarbhering** [reply]
My laptop camera is bad, rear camera from phone deosnt work on github... feels bad man.

**Comment by mrysan** [comment]
It's nearly August and this bug is still not fixed!!!

**Comment by mrysan** [comment]
Here is how you can do it. Use your laptop camera and pull up a picture of your submission/documentation on your phone. Be in a dark/dim room and put your phone brightness on max. Put it up to the laptop camera and it should look pretty good!

**Comment by wr7thful** [comment]
@github pls fix this issue :D

**Comment by wsong83** [comment]
This exact problem is still not fixed!

**Comment by farhan-momin** [comment]
A day ago I applied for Github Education. I faced the same issue of back camera not working.
i tried to take photo with front camera but image was not good. 
Out of frustration i started repeatedly taping the front and back camera options alternatively, and all of a sudden the back camera worked!
i click an image and applied, and i got approved.

Device: android phone
Browser used: Chrome

also, if your proof image isnt coming out to be good, there is some guidance on #111352 to have a better chance with even front cam if back is still not working

**Focus on point 2 of screenshot**

![I can only use my Front Camera to take a photo](https://github.com/user-attachments/assets/d83729b2-6358-453c-b945-0c284ba0808c)

**Comment by buzaiqizhong** [comment]
I'm having the same issue, my Android phone won't enable the rear camera, even though I switched to a different browser (edge, firefox, huawei broswer). 

Next, I also tried to take a virtual camera to get a picture of the screen, but the github webpage didn't respond to the camera.

Finally, I thought of borrowing the camera from my PC, but the photo was distorted and blurry, and I couldn't recognize the content of the picture. 

Hopefully this bug can be solved, or an image upload channel can be provided. Thank you

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)

**Comment by raufzer** [comment]
2024 - Sept 
@github the problem not solved yet , we can't flip between the front and the back cameras

**Comment by abhishek213-alb** [comment]
These guys are so bad they aren't fixing

**Reply by raufzer** [reply]
unbelievable , the issue not been solved from March 2024

**Reply by theProject** [reply]
I'd be happy to fix it so I could submit this proof already, or even let me email it or something as an attachment?  It's not like all programmers and engineers are hackers🪀

**Comment by abzaidi** [comment]
Oct - 2024
My developer pack has expired, and I get rejected every time I take a photo with my front camera while reapplying, simply because this problem has still not been fixed.

**Comment by byrmsh** [comment]


**Reply by saberBenhamda0** [reply]
i know i'm late but i found a solution install opera browser and only give permission to use back camera and there you go , it work !!!

**Comment by saberBenhamda0** [comment]
i know i'm late but i found a solution install opera browser and only give permission to use back camera and there you go , it work !!!

**Comment by Lawrenceku** [comment]
The only way I know to circumvent this issue right now is to use the Opera browser and enable the back camera—it works like a charm.

**Reply by 271314vutbr** [reply]
Thank you so much. The only workaround that works! Can confirm Opera is able to select the back camera on android. It shows a pop up menu that lets you select it directly in the browser rather than github's site's non functioning dropdown menu.

**Comment by HeitorAugustoLN** [comment]
Same here, I just could use the back camera to submit it, by using Opera Browser on my android.

**Comment by marxo112** [comment]
As an iOS user (iPhone 15 in the Safari browser) I can also confirm the problem, the front camera works perfectly with the student ID card option, but the rear camera does not.

**Comment by Abzaek** [comment]
December 6, the issue still exists. I have tried using opera and it doesn't work

**Reply by theProject** [reply]
P pls

**Comment by CihadCengiz** [comment]
I got it to work by revoking the website's camera access while the front camera was selected, then re-enabling camera access and switching to the back camera. Unfortunately, this process has to be repeated each time you take a picture.

**Reply by raufzer** [reply]
For me, it’s still the same issue from 2024 sept , only face camera works on my PC or even on my phone 
disappointed

**Reply by ashrafahyanaju782-beep** [reply]
> I got it to work by revoking the website's camera access while the front camera was selected, then re-enabling camera access and switching to the back camera. Unfortunately, this process has to be repeated each time you take a picture.

**Comment by raufzer** [comment]
it's been about 9 months and the issue not solved yet

**Comment by AlexanderHD27** [comment]
Yeah, no fix after 11 months. I'm trying to get re-verfied, it's incredible annoying.
I faced this issue under IOS for IPad, Linux and Android
